src/test/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/AmsDashboardGenerateJobTest.java

```diff
@@ -137,7 +124,7 @@ void positiveGenerateDashboardFile() {
             } catch (IOException e) {
                 throw new RuntimeException(e);
             }
-        }, times(1));
+        }, times(0));
```

Comment:
If we no longer write to the file we can delete this step

Reply:
Updated